src/roktManager.ts

```diff
@@ -4,19 +4,20 @@ export interface IRoktPartnerAttributes {
 }
 
 // https://docs.rokt.com/developers/integration-guides/web/library/select-placements-options
-export interface ISelectPlacementsOptions {
+export interface IRoktSelectPlacementsOptions {
     attributes: IRoktPartnerAttributes;
     identifier?: string;
 }
 
-export interface ISelection {
-    placementId?: string;
-    status?: string;
-    error?: string;
+interface IRoktPlacement {}
+
+export interface IRoktSelection {
```

Comment:
i'd add links for each interface that we're pulling for easier searching in the future
https://docs.rokt.com/developers/integration-guides/web/library/selection

Reply:
maybe there should be a jira ticket for ensuring these interfaces eventually get fully typed since they aren't all here right now?